src/RealizationOrder.cpp

```diff
@@ -163,9 +163,15 @@ void populate_fused_pairs_list(const string &func, const Definition &def,
                    func, stage_index, fuse_level.var().name());
     if (fuse_level.stage_index() == 0) {
         parent.definition().schedule().fused_pairs().push_back(pair);
+        for (int i = 0; i < parent.definition().specializations().size(); i++) {
```

Comment:
style nit: `for (const auto s : parent.definition().specializations())` might be a better choice here

Reply:
Done